app/src/main/java/org/piwigo/io/repository/PreferencesRepository.java

```diff
@@ -29,6 +29,7 @@
     public static final String KEY_ACTIVE_ACCOUNT = "active_account";
     public static final String KEY_PREF_PHOTOS_PER_ROW = "photos_per_row";
     public static final String KEY_PREF_DOWNLOAD_SIZE = "download_size";
+    public static final String KEY_PREF_COLOR_PALETTE = "color_palette";
```

Comment:
good, to have that here, so also use it! And the default should also be handled in the PreferenceRepository and not "everywhere" in the places we need the setting.

Reply:
See https://github.com/Piwigo/Piwigo-Android/pull/201/commits/12a40da734260be6add46bed7c180c6d3cd1dc9c and https://github.com/Piwigo/Piwigo-Android/pull/201/commits/fb30c91cd9c61c7694be4a08d9b5711fb98083fe